spec/features/hiring_staff_can_publish_a_vacancy_spec.rb

```diff
@@ -464,6 +464,16 @@
     end
 
     context '#publish' do
+      scenario 'adds the current user as a contact for feedback on the published vacancy' do
+        current_user = User.find_by(oid: session_id)
+        vacancy = create(:vacancy, :draft, school: school)
+
+        expect(PublishVacancy).to receive(:new).with(vacancy, current_user).and_call_original
```

Comment:
I'd prefer this feature spec not to mock out this method.  I'd prefer it to be asserting against what you actually care about, i.e. the `publisher_user_id` column being populated.